### PR TITLE
Add Swift Package files

### PR DESCRIPTION
**Pull request by ale-gen**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "JustRideSDK",
```
Comment (adil-hussain-84) [inline Package.swift:7]: We'll have to leave this with a big 'R' so existing SDK Integrators who switch over to getting the Justride SDK via Swift Package Manager don't encounter compilation errors in their code.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ale-gen/RNCryptor.git", exact: "5.1.0"),
+        .package(url: "https://github.com/ale-gen/MarqueeLabel.git", exact: "4.3.2"),
+        .package(url: "https://github.com/ale-gen/ZipArchive.git", exact: "2.4.3"),
+        .package(url: "https://github.com/ale-gen/zxingify-objc.git", exact: "3.6.7")
+    ],
+    targets: [
+        .target(
+            name: "JustRideSDKTarget",
```
Comment (adil-hussain-84) [inline Package.swift]: Please name this "JustRideSDKWrapper" 🙏

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            dependencies: [
+                .target(
+                    name: "JustRideSDK",
+                    condition: .when(platforms: [.iOS])
```
Comment (adil-hussain-84) [inline Package.swift:28]: If you remove this line, does anything break?

File: JustRideSDKWrapper/dummy.m [hunk 4]
```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright 2023 Masabi Ltd
+ */
+
+// Swift Package Manager needs at least one header to prevent a warning. See
```
Comment (adil-hussain-84) [inline JustRideSDKWrapper/dummy.m]: Please change the opening part of this comment to: "A Swift Package Manager target needs..."

File: JustRideSDKWrapper/include/dummy.h [hunk 5]
```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright 2023 Masabi Ltd
+ */
+
+// Swift Package Manager needs at least one source file.
```
Comment (adil-hussain-84) [inline JustRideSDKWrapper/include/dummy.h]: Please change the opening part of this comment to: "A Swift Package Manager target needs..."

File: Package.swift (hunk 3 above)
Comment (ale-gen) [reply Package.swift:28]: This line just explicitly tells for which platforms dependency should be applied

Comment (adil-hussain-84) [reply Package.swift:28]: Okay, let's discuss after standup. I'll merge this PR in as it is for now.

**Review by adil-hussain-84 — APPROVED** [review]

**Pull request merged**